src/main/kotlin/no/nav/medlemskap/domene/Datagrunnlag.kt

```diff
@@ -9,7 +9,8 @@ data class Datagrunnlag (
         val medlemskap: List<Medlemskap> = listOf(),
         val arbeidsforhold: List<Arbeidsforhold> = listOf(),
         val oppgaver: List<Oppgave> = listOf(),
-        val dokument: List<Journalpost> = listOf()
+        val dokument: List<Journalpost> = listOf(),
+        var ytelse: Ytelse?
```

Comment:
Hvorfor nullable? Du behandler den som om den ikke er det i koden, veldig mange !! der..?

Reply:
Det var for å slippe å oppdatere alle json-filene i test. Men fjernet nullable nå og heller oppdaterte tester til å ha med sykepenger som ytelse i datagrunnlag.

Reply:
Ah, skjønner. Ja, det er bra!